src/mlpack/methods/ann/loss_functions/mean_squared_logarithmic_error.hpp

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file mean_squared_logarithmic_error.hpp
+ * @author Saksham Rastogi
+ *
+ * Definition of the mean squared logarithmic error function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LOSS_FUNCTION_MEAN_SQUARED_LOGARITHMIC_ERROR_HPP
+#define MLPACK_METHODS_ANN_LOSS_FUNCTION_MEAN_SQUARED_LOGARITHMIC_ERROR_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The mean squared logarithmic error performance function measures the network's
+ * performance according to the mean of squared logarithmic errors.
+ *
+ * @tparam InputDataType Type of the input data (arma::colvec, arma::mat,
+ *         arma::sp_mat or arma::cube).
+ * @tparam OutputDataType Type of the output data (arma::colvec, arma::mat,
+ *         arma::sp_mat or arma::cube).
+ */
+template <
```

Comment:
Minor gripe, the `typename` should immediately follow the `<`, also no need for a new line at the end before the closing `>`!